src/service.ts

```diff
@@ -44,15 +44,11 @@ async function resolveConfig(
 }
 
 async function resolvePrettier(cwd: string): Promise<typeof Prettier> {
-  let prettier: typeof Prettier;
-
   try {
-    prettier = await import(require.resolve("prettier", { paths: [cwd] }));
+    return import(require.resolve("prettier", { paths: [cwd] }));
```

Comment:
Is that the fix you talked about in the PR https://github.com/fsouza/prettierd/pull/167 ?

I am quite surprised that this works properly, if you return without awaiting then you'll never get into the catch 🤔 

I would've expected this:

```typescript
return await import(require.resolve("prettier", { paths: [cwd] }));
```